nix/mkDevShell.nix

```diff
@@ -25,6 +25,7 @@ let
   stableShell = pkgs.mkShell {
     packages = devDependencies ++ [ bkyAsStable ];
     shellHook = ''
+      set -e
```

Comment:
Even though it is unlikely that the echo command will fail, I added the exit condition to the shell hook so that both hooks have a similar termination behavior